js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/vectorIndex/useCreateVectorIndex.ts

```diff
@@ -0,0 +1,86 @@
+import * as Yup from "yup";
+import { commonFieldsMap, commonSchema } from "../IndexFieldsHelper";
+import { useCreateIndex } from "../useCreateIndex";
+
+export const INITIAL_VALUES = {
+  type: "vector",
+  fields: commonFieldsMap.fields.initialValue,
+  name: commonFieldsMap.name.initialValue,
+  params: {
+    metric: "",
+    dimension: undefined,
+    nLists: undefined,
+    defaultNProbe: 1,
+    trainingIterations: 25,
+  }
+};
+
+export const FIELDS = [
+  commonFieldsMap.fields,
+  commonFieldsMap.name,
+  {
+    label: "Metric",
+    name: "params.metric",
+    type: "creatableSingleSelect",
+    isRequired: true,
+    options: [
+      { label: "Cosine", value: "cosine" },
+      { label: "L2 (Euclidean)", value: "l2" }
+    ],
+    tooltip: "Distance metric used for similarity search. Choose between cosine similarity or Euclidean distance."
+  },
+  {
+    label: "Dimension",
+    name: "params.dimension",
+    type: "number",
+    isRequired: true,
+    tooltip: "The dimensionality of the vector embeddings. The indexed attribute must contain vectors of this length."
+  },
+  {
+    label: "Number of Lists (nlists)",
+    name: "params.nLists",
+    type: "number",
+    isRequired: true,
+    tooltip: "The number of centroids (clusters) to divide the vector space into. A higher value improves recall but increases indexing time. Suggested value: ~15 × N, where N is the number of documents."
+  },
+  {
+    label: "Default N Probe",
```

Comment:
If we don't use the internal name (`defaultNProbe`) here, then we can as well write out the N
```suggestion
    label: "Default Number of Probes",
```